app/views/projects/index/_project_index.html.erb

```diff
@@ -0,0 +1,39 @@
+<p>You are signed in with the email address <%= current_user.email %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <h1 class="govuk-heading-l">
+      <%= t("project.index.title.all") %>
+    </h1>
+
+    <% if policy(:project).new? %>
+      <p><%= link_to t("project.index.new_button.text"),
+               new_project_path,
+               class: "govuk-button",
+               data: {module: "govuk-button"} %></p>
+    <% end %>
+
+    <%= render partial: "shared/project_index_sub_navigation" %>
+
+    <%- if @projects.empty? %>
```

Comment:
We haven't used these `-` anywhere else, I'm not sure what they do, or do they just do the same as omitting the `-`?

Reply:
You know what I think these are quite old erb style and I haven't used Ruby for a year so I'm out of date 😆